tests/test_chembridge.py

```diff
@@ -208,6 +208,9 @@ def test_get_dipole_moments() -> None:
     molobj = tasks.generate_conformers(molobj, n_conformers=n_conformers)
 
     moments = chembridge.get_dipole_moments(molobj)
+
+    print(moments)
```

Comment:
is the print statement still needed here?

Reply:
Print statements in tests are fine, as they only get printed out on test failures